firebase.json

```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
+    "public": "build",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
+      {
+        "source": "**",
+        "destination": "/index.html"
+      }
```

Comment:
I think for now I'd prefer to not do rewriting here. And let any requests to missing resources just 404. If we add retrieving data from the URL, we can think about it then.